static/app/components/group/externalIssuesList/useExternalIssueData.tsx

```diff
@@ -101,6 +101,10 @@ export default function useExternalIssueData({group, event, project}: Props) {
   };
 
   const renderPluginIssues = (): ExternalIssueComponent[] => {
+    if (!group.pluginIssues) {
+      return [];
+    }
```

Comment:
This might not be strictly needed. 
The `group.pluginIssues?.map` call below will return an array or undefined in a safe way.
And that undefined will get splat together with other arrays in the part:
```
 const actions = [
    ...renderSentryAppIssues(),
];
```

Reply:
ah yeah you're right, this was a change from before the expand calls were working. will remove